include/identity.php

```diff
@@ -603,6 +604,13 @@ function advanced_profile(&$a) {
 		if($txt = prepare_text($a->profile['work'])) $profile['work'] = array( t('Work/employment:'), $txt);
 
 		if($txt = prepare_text($a->profile['education'])) $profile['education'] = array( t('School/education:'), $txt );
+	
+		//show subcribed forum if it is enabled in the usersettings
+		if (feature_enabled($uid,'forumlist_profile')) {
+			require_once('include/forums.php');
```

Comment:
> Could you move the include to the header? I think this is cleaner.

Should I move it to the top of the file or the top of function advanced_profile(&$a)?